packages/chain-adapters/package.json

```diff
@@ -22,7 +22,7 @@
     "@shapeshiftoss/unchained-client": "workspace:^",
     "@shapeshiftoss/utils": "workspace:^",
     "@solana/spl-token": "^0.4.9",
-    "@solana/web3.js": "^1.95.3",
+    "@solana/web3.js": "^1.98.0",
```

Comment:
Welp, that answers half of my q. above!
Nice one on aligning versions big @0xApotheosis 🐐

Reply:
I think we can even remove this line if we add the lib globally now?

Reply:
Strongly advising against that, packages should leverage their own version (even though now aligned with web).
Not being explicit in versions here would not necessarily mean we use web's (we may or may not depending on current setup) and could result in packages consuming transient deps which at best could produce bugs and at worst could introduce vulnerabilities.

Reply:
And generally speaking, a package using a dep and not requiring it explicitly in package.json but rather relying on consumers having it installed is a blatant bug, even though it *may* work here as the only real consumer of this is web (though, nothing blocking any other package from using `chain-adapters` if they wanted to)

Reply:
I was about to say "remove it from the global package.json then" but I finally realized we aren't really using a proper mono-repo structure but this package.json is the web app 😆 

Looks fine then